clientcontroller/babylon/babylon.go

```diff
@@ -268,22 +290,15 @@ func (bc *BabylonController) UnjailFinalityProvider(fpPk *btcec.PublicKey) (*typ
 		return nil, err
 	}
 
-	return &types.TxResponse{TxHash: res.TxHash, Events: res.Events}, nil
-}
-
-// QueryFinalityProviderSlashedOrJailed - returns if the fp has been slashed, jailed, err
-func (bc *BabylonController) QueryFinalityProviderSlashedOrJailed(fpPk *btcec.PublicKey) (bool, bool, error) {
-	fpPubKey := bbntypes.NewBIP340PubKeyFromBTCPK(fpPk)
-	res, err := bc.bbnClient.QueryClient.FinalityProvider(fpPubKey.MarshalHex())
-	if err != nil {
-		return false, false, fmt.Errorf("failed to query the finality provider %s: %w", fpPubKey.MarshalHex(), err)
+	if res == nil {
+		return &types.TxResponse{}, nil
```

Comment:
how could this case happen?

Reply:
This was actually introduced at https://github.com/babylonlabs-io/finality-provider/pull/124 by you :sweat_smile: Do you remember any context?